getting-started/scaling-out.md

```diff
@@ -0,0 +1,144 @@
+# Scaling out using Distributed Hypertables
+
+TimescaleDB supports multi-node clustering by leveraging the hypertable and chunk primitives.
+Hypertables are used to handle large amount of data by breaking it up
+in smaller pieces (chunks), allowing operations to execute efficiently. When
+the amount of data is expected to be beyond what a single machine can handle,
+you can distribute the data chunks over several machines by using
+*distributed hypertables*.
+Distributed hypertables are similar to normal hypertables, but they
+add an additional layer of hypertable partitioning by distributing chunks
+across *data nodes*.
+
+In the multi-node topology ([architecture][]), all nodes are TimescaleDB instances.
+The data are distributed and stored in TimescaleDB instances, called data nodes.
+The distributed data are accessed through distributed hypertable
+on another TimescaleDB instance, called the *access node*.
+The access node is the entry point for any access to data in the cluster.
+
+## Working with Data Nodes
+
+Data nodes act as the containers for the hypertable chunks and are
+necessary to create distributed hypertables. Data nodes are
+added to the current database using [`add_data_node`][add_data_node]
+and removed using [`delete_data_node`][delete_data_node].
+
+Note that:
+
+* Data nodes are added as objects locally and just connect to an
+  existing database server.
+
+* You should already have a running database server on some host.
+
+When creating the data node, you should:
+
+* Provide a (local) name to use when referring to the data node from
+  this database.
+
+* Provide the host where the hypertable partition for the distributed
+  hypertable should be stored.
+
+* Ensure that the user used for connecting to the data node has
+  `CREATEDB` permissions or be a superuser. This is necessary since
+  `create_distributed_hypertable` expect to be able to create a
+  database on the remote data node.
+
+```sql
+SELECT add_data_node('node1', host => 'dn1.example.com');
+SELECT add_data_node('node2', host => 'dn2.example.com');
```

Comment:
I'd really like to see us specify password in our examples, just because of how easy it is to mess up your cluster if you're running as a superuser and don't specify the password (though we should really fix this by making the code more robust).

Reply:
Added by copying from the tutorial doc.